src/coap_blockwise.c

```diff
@@ -339,7 +344,9 @@ static void on_block_rcvd(struct golioth_client *client,
     ctx->is_last = is_last;
     memcpy(ctx->buffer, payload, payload_size);
     ctx->rcvd_bytes = payload_size;
-    ctx->status = response->status;
+    ctx->status = status;
+    ctx->coap_rsp_code.status_class = coap_rsp_code->status_class;
+    ctx->coap_rsp_code.status_detail = coap_rsp_code->status_detail;
```

Comment:
Do we ever use this in the blockwise get operation?

Reply:
We don't use this for a blockwise get so they do not need to be set in this callback.

Reply:
Removing line 347 caused block download to immediately fail for espidf. The status is passed back from the context to the falling function and still needs to be assigned here. However, the coap code is not used and doesn't need to be assigned.